service/src/main/java/com/ge/predix/acs/attribute/connector/management/AttributeConnectorException.java

```diff
@@ -0,0 +1,18 @@
+package com.ge.predix.acs.attribute.connector.management;
+
+public class AttributeConnectorException extends RuntimeException {
```

Comment:
It's not a true checked exception since we don't have throws in service methods.  Also, converting it to checked exception will not work with a stream used to validate adapter configuration:
connector.getAdapters().parallelStream().forEach(adapter -> {
  validateAdapterEntityOrFail(adapter);
});
I would rather keep this exception as runtime.